dev/com.ibm.websphere.org.eclipse.microprofile.faulttolerance.1.0/bnd.bnd

```diff
@@ -24,7 +24,10 @@ Export-Package: \
   org.eclipse.microprofile.faulttolerance;version=1.0, \
   org.eclipse.microprofile.faulttolerance.exceptions;version=1.0
 
-Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version>=1.8))"
+Include-Resource: \
+  @${repo;org.eclipse.microprofile.fault-tolerance:microprofile-fault-tolerance-api;1.0.0}
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
```

Comment:
why not >=

Reply:
`=` is correct here. This capability specifies that it requires something _compatible_ with 1.8.

Future Java versions should specify the 1.8 capability as long as they remain compatible with 1.8.

See OSGi Core v6 - 8.2

Reply:
Because apparently that is wrong ... this header specifies a minimum version.

Reply:
ok, thanks for the explanation @tevans78 @Azquelt !